chat/SettingsView.vue

```diff
@@ -276,6 +276,35 @@
         </label>
       </div>
 
+      <h5>Draft Messages</h5>
+
+      <div class="form-group">
+        <label class="control-label" for="horizonCacheDraftMessages">
+          <input
+            type="checkbox"
+            id="horizonCacheDraftMessages"
+            v-model="horizonCacheDraftMessages"
+          />
+          Automatically save and restore in-progress messages while using
```

Comment:
Mind converting these two descriptions (and the title) to use the localization system instead? We'll be doing this across the board later on, so we might as well not increase the workload for the future.